nix/services/memcached.nix

```diff
@@ -0,0 +1,67 @@
+# Based on https://github.com/cachix/devenv/blob/main/src/modules/services/memcached.nix
+{ pkgs
+, lib
+, name
+, config
+, ...
+}:
```

Comment:
Could you put this in single line to be consistent with the formatting of the rest of the project?

Reply:
Will do. I remember seeing treefmt formatting it in a pre-commit-hook, must have been dreaming.

Reply:
It could have been your editor doing an auto formatting on save.

Reply:
Yes, my editor definitely did that, but I saw treefmt changing 4 files. But now when I manually call it, it doesn't do anything. Will have to manually revert those changes.
```
% just fmt
treefmt
0 files changed in 5ms (found 109, matched 54, cache misses 0)
```